yletunnus/backends.py

```diff
@@ -24,7 +36,10 @@ def auth_params(self, state=None):
 
     def access_token_url(self):
         app_id, app_key = self.setting('APP_ID'), self.setting('APP_KEY')
-        return self.ACCESS_TOKEN_URL + '?app_id=%s&app_key=%s' % (app_id, app_key)
+        return self._get_url('token?app_id=%s&app_key=%s' % (app_id, app_key))
```

Comment:
I know this string formatting is as is. But could be changed to more modern?

```python
'token?app_id={}&app_key={}'.format(app_id, app_key)
# or
f'token?app_id={app_id}&app_key={app_key}'
```

Reply:
Fixed in 9df56232a4377d4d370197884f79abb013498df6